src/features/surveys/hooks/useCreateSurveySubmission.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  ZetkinSurveyFormStatus,
+  ZetkinSurveyQuestionResponse,
+} from 'utils/types/zetkin';
+import { useApiClient } from 'core/hooks';
+
+export type ZetkinSurveyCallSubmission = {
+  responses: ZetkinSurveyQuestionResponse[];
+  signature: number;
+};
+
+export default function useCreateSurveySubmission(
+  orgId: number,
+  surveyId: number
+) {
+  const apiClient = useApiClient();
+
+  const addSubmission = (
```

Comment:
The name of the hook and the name of the function it returns seem unnecessarily different to me. I would maybe call the hook `useSubmitSurveySubmission()` and the function `submitSurveySubmission()`, or something like that. If you prefer using CRUD terminology, use `create` in both places.